### ISSUE TITLE
Is there are some read me to show how to run this code?

### ISSUE THREAD

**Issue by Gatsby23** [EpicDuckPotato/stereo_vo#2]

**Comment by EpicDuckPotato** [comment]
Sorry, this was an old personal project that I don't maintain, but I added a README. Let me know if you're able to get it running.

**Comment by Gatsby23** [comment]
> Sorry, this was an old personal project that I don't maintain, but I added a README. Let me know if you're able to get it running.

Thank you very much, I'll test it right now.

**Comment by Gatsby23** [comment]
![Screenshot from 2021-03-30 10-57-08](https://user-images.githubusercontent.com/18000675/112927106-0951e680-9147-11eb-8e9b-bc668dd0f7b6.png)

here is the running status, it haven't run any process code

**Comment by Gatsby23** [comment]
> > Sorry, this was an old personal project that I don't maintain, but I added a README. Let me know if you're able to get it running.
> 
> Thank you very much, I'll test it right now.

OK, it can run right now. Thanks for your patient replay, thank you very much!

**Comment by Gatsby23** [comment]
> Sorry, this was an old personal project that I don't maintain, but I added a README. Let me know if you're able to get it running.

Sorry for troubling you, the path in rviz seems like kitti ground gruth path. It seems that it hasn't run the vo system.

**Comment by EpicDuckPotato** [comment]
The kitti ground truth path should be in green, and the vo path should be in red. Is this not what you're seeing?

**Comment by Gatsby23** [comment]
Yes, I haven't see the red line, later, I'll send the wrong message here.
